third_party/islet/setup.sh

```diff
@@ -29,14 +29,15 @@ ISLET_INC="$HERE/include"
 ISLET_LIB="$HERE/lib"
 
 TARGET_HDR="$ISLET_SDK/include/islet.h"
-TARGET_LIB="$ISLET/out/x86_64-unknown-linux-gnu/debug/libislet_sdk.so"
+TARGET_LIB="$ISLET/out/x86_64-unknown-linux-gnu/debug/libislet_sdk.a"
```

Comment:
This change from `.so` to `.a` seems to be like build-artifact issue... 

So, I'm not going to worry about this too much, expecting that as this works in CI, things are ok.